<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>667</width>
    <height>329</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Your message:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="chatLineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="sendToAllButton">
        <property name="text">
         <string>send to all</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sendPrivateButton">
        <property name="text">
         <string>send private</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="logoutButton">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>CrossCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {background-color:#e80000;}</string>
        </property>
        <property name="text">
         <string>logout</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <widget class="QTextBrowser" name="publicBrowser"/>
      <widget class="QTextBrowser" name="privateBrowser"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>667</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuType_here">
    <property name="title">
     <string>menu</string>
    </property>
    <addaction name="actionclose_client"/>
   </widget>
   <addaction name="menuType_here"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionclose_client">
   <property name="text">
    <string>close client</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
